package.json

```diff
@@ -40,7 +40,7 @@
     "xmldom": "0.1.22",
     "logops": "1.0.0-alpha.7",
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master",
```

Comment:
Maybe this PR is not neded if task/updateCoap is merged first in lwm2m-node-lib? Could you elaborate on task/updateCoap merging plan?

Reply:
We need this PR to update the version of lwm2m-node-lib in https://github.com/telefonicaid/lightweightm2m-iotagent/blob/master/package.jsonL#43. As you can see, now it is fixed to 0.5.0.

Reply:
But the change is

```
-    "lwm2m-node-lib": "0.5.0",
+    "lwm2m-node-lib": "git://github.com/dcalvoalonso/lwm2m-node-lib.git#task/updateCoap",
```

and I understand it should be:

```
-    "lwm2m-node-lib": "0.5.0",
+    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master",
```

specially now that task/updateCoap has been merged into master in PR https://github.com/telefonicaid/lwm2m-node-lib/pull/126

Or maybe I'm missing something? :)

Reply:
You're more than right. This was just temporary to demonstrate that https://github.com/telefonicaid/lwm2m-node-lib/pull/126 did not affect the agent functionality and for this reason, I marked the PR as [WIP].

I will push a definite PR and remove the [WIP]! ;)

Reply:
Fixed by 61b6fe580d8a1f1b1479443c553a48fbd0215db6